### DISCUSSION TITLE
Sujet / Thema 2 –– AV frei verfügbar in allen Kantonen / MO disponible dans tous les cantons

### DISCUSSION THREAD

**Discussion by Fijord** [GeoUnconference/discussions#18] [Ideen / Idées]
Es wäre einen grossen Vorteil, wenn wirklich alle Kantonen mitmachen und nicht zum Beispiel ein Loch in der Mitte der Schweiz entsteht.

**Comment by hinjud** [comment]
Was ist mit "frei verfügbar" gemeint? Über geodienste.ch (https://geodienste.ch/services/av) können mit Ausnahme des Tessins (im Aufbau) alle AV-Datensätze bezogen werden.

**Reply by davidoesch** [reply]
@https://github.com/Fijord benötigt die Daten in Webmercator und als WMTS

**Reply by hinjud** [reply]
EPSG:3857 ist gemäss Capabilities verfügbar. Wunsch für WMTS als Bereitstellungsformat könnte als Feedback (https://geodienste.ch/feedback) deponiert werden.

**Reply by davidoesch** [reply]
@Fijord : kannst du den Webmercator request via https://geodienste.ch/feedback platzieren?

**Reply by Fijord** [reply]
Vielen Dank für eure Unterstützung. Wir bräuchten einen WMTS ohne Versatz. Habe es auf geodienste.ch als Feedback platziert.
Zusätzlich meinte ich, dass die AV bei allen Dienste öffentlich zugänglich sind. Wie zum Beispiel die AV der Zentralschweiz und der West- und Ostschweiz, siehe [https://s.geo.admin.ch/91edf397a4](url)

**Reply by zigertiger** [reply]
Bei dieser Thematik spielen (leider) noch Gebührenverordnungen und auch unterschiedliche Nutzungsbestimmungen hinein. Beispiel Kt. GL: der eigentlich perverse Grund dafür, dass keine AV-Daten des Kt GL via https://s.geo.admin.ch/91edf397a4 direkt verfügbar sind, ist, dass die OpenData-Nutzungsbestimmungen des Kt. GL _offener_ sind als jene, die von der swisstopo für die AV-Daten gebastelt wurden. Das führt dazu, dass die AV-Daten nicht via geo.admin.ch publiziert werden _dürfen_, weil diese Bestimmungen zu restriktiv sind.

Lösung (abgesehen von noch bestehenden Gebührenmodellen): Open Definition (https://opendefinition.org/) anwenden, keine _zwingende_ Quellenangabe mehr (das kann ohnehin nicht kontrolliert werden). Und vor allem: KEINE EIGENEN QUASI-OPEN-DATA-BESTIMMUNGEN ERFINDEN!

**Reply by davidoesch** [reply]
die amtliche vermessung wird diesbezüglich angegangen

**Reply by schenardi** [reply]
Was ich mir auf Geodienste.ch zumindest für die AV Daten wünschen würde: nicht nur das "plumpe" Hinterlegen der Datenbestimmungen der jeweiligen Kantone sondern zumindest eine Tabelle als Zusammenfassung, was ich aktuell mit den Daten darf (und was grade nicht). 
* Weitergabe an Dritte ja/nein
* kommerzielle Nutzung ja/nein
* Online Dienst mit diesen Daten öffentlich bereitstellen ja/nein
* etc.

Wäre allenfalls auch ein erster simpler Approach für eine Vereinheitlichung > etwas weg von der Juristerei zum praktischen Nutzen...

**Comment by ratnanil** [comment]
> Was ist mit "frei verfügbar" gemeint? 

Um es nochmals zu verdeutlichen: Als WFS .shp oder .gpkg "Frei erhältlich" sind aktuell folgende Kantone: 

![image](https://user-images.githubusercontent.com/12532091/168536645-8e5ee47d-6604-484f-bf1f-9447d1c2c616.png)

Für nachstehende Kantone ist noch eine Registrierung erforderlich. Ich habe nachgefragt: für gewisse Kantone belaufen sich die Kosten für einen Bezug auf fast **50'000 CHF** 

![image](https://user-images.githubusercontent.com/12532091/168536980-0de4ea39-df9b-4ebf-9f9d-01129920044d.png)

**Comment by rastrau** [comment]
During GeoUnconference 3, a group including @fuerstenberger have discussed this topic (in conjunction with [topic 3](https://github.com/GeoUnconference/discussions/discussions/16)). The questions have been broadened and encompass how to stay in touch the end users' needs and how to keep data models flexible enough for various and evolving requirements.

These are the insights from the group: 

![IMG_20220505_161614](https://user-images.githubusercontent.com/4510815/170704217-4bd4f548-acf2-4d49-bf71-13cb4b625047.jpg)
![1](https://user-images.githubusercontent.com/4510815/170704267-357b70dd-25d9-4975-83f2-04d93adaaf34.jpg)
